src/server/search/aggregator.cc

```diff
@@ -11,10 +11,10 @@ namespace dfly::aggregate {
 namespace {
 
 struct GroupStep {
-  PipelineResult operator()(std::vector<DocValues> values) {
+  PipelineResult operator()(PipelineResult result) {
```

Comment:
PipelineResult contains more than result.values but you use only this field, maybe you don't need to change the type of parameter